### ISSUE TITLE
dicttoxml(None) and dicttoxml("name") break in 1.7

### ISSUE THREAD

**Issue by michael-k** [quandyfactory/dicttoxml#53]
In 1.6.6:

``` python
In [1]: from dicttoxml import dicttoxml
```

``` python
In [2]: dicttoxml(None)           
Out[2]: b'<?xml version="1.0" encoding="UTF-8" ?><root><item type="str"></item></root>'
```

``` python
In [3]: dicttoxml("name")
Out[3]: b'<?xml version="1.0" encoding="UTF-8" ?><root><item type="str">name</item></root>'
```

In 1.7.4:

``` python
In [1]: from dicttoxml import dicttoxml
```

``` python
In [2]: dicttoxml(None)                
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-2-d2c15ddde005> in <module>()
----> 1 dicttoxml(None)

/usr/local/lib/python3.4/dist-packages/dicttoxml.py in dicttoxml(obj, root, custom_root, ids, attr_type, item_func, cdata)
    391         addline('<%s>%s</%s>' % (
    392         custom_root,
--> 393         convert(obj, ids, attr_type, item_func, cdata, parent=custom_root),
    394         custom_root,
    395         )

/usr/local/lib/python3.4/dist-packages/dicttoxml.py in convert(obj, ids, attr_type, item_func, cdata, parent)
    184 
    185     if obj is None:
--> 186         return convert_none(item_name, '', attr_type, cdata)
    187 
    188     if isinstance(obj, dict):

/usr/local/lib/python3.4/dist-packages/dicttoxml.py in convert_none(key, val, attr_type, attr, cdata)
    361 
    362     if attr_type:
--> 363         attr['type'] = get_xml_type(val)
    364     attrstring = make_attrstring(attr)
    365     return '<%s%s></%s>' % (key, attrstring, key)

TypeError: 'bool' object does not support item assignment
```

``` python
In [3]: dicttoxml("name")              
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-3-0cf4e76abe5a> in <module>()
----> 1 dicttoxml("name")

/usr/local/lib/python3.4/dist-packages/dicttoxml.py in dicttoxml(obj, root, custom_root, ids, attr_type, item_func, cdata)
    391         addline('<%s>%s</%s>' % (
    392         custom_root,
--> 393         convert(obj, ids, attr_type, item_func, cdata, parent=custom_root),
    394         custom_root,
    395         )

/usr/local/lib/python3.4/dist-packages/dicttoxml.py in convert(obj, ids, attr_type, item_func, cdata, parent)
    175 
    176     if isinstance(obj, numbers.Number) or type(obj) in (str, unicode):
--> 177         return convert_kv(item_name, obj, attr_type, cdata)
    178 
    179     if hasattr(obj, 'isoformat'):

/usr/local/lib/python3.4/dist-packages/dicttoxml.py in convert_kv(key, val, attr_type, attr, cdata)
    331 
    332     if attr_type:
--> 333         attr['type'] = get_xml_type(val)
    334     attrstring = make_attrstring(attr)
    335     return '<%s%s>%s</%s>' % (

TypeError: 'bool' object does not support item assignment
```

**Comment by quandyfactory** [comment]
This will be fixed in version 1.7.9, coming soon.

**Comment by quandyfactory** [comment]
This is now fixed in v1.7.13.
